mix.exs

```diff
@@ -21,9 +34,9 @@ defmodule Tee.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      # {:dep_from_hexpm, "~> 0.3.0"},
-      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"},
-      {:membrane_core, "~> 0.2.0"},
+      {:membrane_core, "~> 0.3.0"},
```

Comment:
I cannot comment on the not edited lines, but please remove `:logger` from `extra_applications`. You can also get rid of the generated comments

Reply:
I deleted the whole function as most likely it is not necessary to be here